### DISCUSSION TITLE
slug should use get_page_by_path in some way for heirarchical post types in pods()

### DISCUSSION THREAD

**Discussion by pglewis** [pods-framework/pods#6605] [Ideas]
slug should use get_page_by_path in some way for heirarchical post types in pods()

**Comment by quasel** [comment]
didn't happen above my pay grade ;) but stuff like that isn't really a bug or is it?

**Comment by JoryHogeveen** [comment]
I don't really see how this can be done since the full path isn't stores in the posts table.
